Project/backend/.env.example

```diff
@@ -17,3 +17,6 @@ POSTGRES_PORT=5432
 POSTGRES_HOST=amos-db
 
 JANUS_PORT=8182
+
+# Docker
+DOCKER_COMPOSE=docker-compose
```

Comment:
I don't know if I made a setup mistake with docker but with me the variable was not set and because you check it in the make file for the version I had to set it manually. But that may have to do with my setup. 
```python

DOCKER_COMPOSE != docker compose 1> /dev/null 2> /dev/null && echo docker compose || echo docker-compose
/
```